.github/workflows/ci.yml

```
@@ -218,7 +219,7 @@ jobs:
           save-if: ${{ github.ref == 'refs/heads/master' }}
 
       - name: Run cargo clippy
-        run: cargo custom-clippy # cargo alias to allow reuse of config locally
+        run: cargo clippy # cargo alias to allow reuse of config locally
```

```suggestion
        run: cargo clippy
```

Can you also delete the cargo alias in `.cargo/`? I think we may be able to get rid of the entire file!

---

sure